hack/scripts/kernel.sh

```diff
@@ -0,0 +1,123 @@
+# Should be used for manual validation. Does not check if correct kernel is installed or if installation failed.
+# If used within automation, error check at the end
```

Comment:
Let's add a shebang `#!/bin/bash` and `set -e`

Reply:
Is failing fast worth fighting through false negatives coming from kubectl exec?